sdk/communication/communication-chat/swagger/README.md

```diff
@@ -10,8 +10,8 @@ title: ChatApiClient
 description: Chat Client
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../src/generated
-tag: package-chat-2021-09-07
-require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/communication/data-plane/Chat/readme.md
+tag: package-chat-2023-11-07
+require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/chat-inline-image-ga/specification/communication/data-plane/Chat/readme.md
```

Comment:
I am assuming this whole directory will need to change once the change is pushed up to main?

Reply:
Yeah, I think so, or to a specific commit has.